### ISSUE TITLE
Payments keep counting down after client disconnected

### ISSUE THREAD

**Issue by ronnylov** [LetheanMovement/lethean-vpn#104]
I have noticed in /opt/lthn/var/log/audit.log that payment balance for authids keep counting down to zero after client disconnected. Decided to test this.

Paid 9 LTHN to a provider that charge 0.05 LTHN/min so this should last 3 hours. Disconnected after approximately 10 minutes and the balance kept counting down until zero on exit node.

See attached logs.
[Client_interaction.txt](https://github.com/LetheanMovement/lethean-vpn/files/3232416/Client_interaction.txt)
[Client_audit.log.txt](https://github.com/LetheanMovement/lethean-vpn/files/3232418/Client_audit.log.txt)
[audit.log](https://github.com/LetheanMovement/lethean-vpn/files/3232421/audit.log)
[lthnvpnd_debug.txt](https://github.com/LetheanMovement/lethean-vpn/files/3232424/lthnvpnd_debug.txt)

**Comment by ronnylov** [comment]
This makes the idea of prepaid authids to not work because it will run out of money even if not used. This must be a bug?

**Comment by Mahdi321** [comment]
If you want to add the feature of using remaining credit, please keep your system account-free (as it's now), otherwise there would be worry for users who use your service through a 3rd party vpn, is he/she always started your program with the primary vpn on.
Actually you can do it with payment receipt IDs that are only valid if the client IP is remained same as the one that is mentioned in payment receipt (patent pending (I'm in urgent need of money otherwise I wouldn't add these lame last 2 words at end))

**Comment by limosek** [comment]
This is feature from design, not bug.
Authid starts spending by first session (or just after payment, according to settings) and spending is not paused at any time. It is technically very hard to make spending paused on no traffic.

**Comment by ronnylov** [comment]
The payment "account" is the same as the payment id used. It is possible ti reconnect using paymentid@prividerid/serviceid and continue use it as long as balance is left on thus paymentid. The problem is that money is reduced no matter if connected or not.

**Comment by limosek** [comment]
Yes, for now, it is recommended to use small payments. Recommended way for achieving more persistent authids is to define minimal minutes longer, like 1 month for some service. So uder can use this proxy entire month regardless of usage. But, again, security is playing role. More authids means more privacy.
